dsls/sql/src/test/java/org/apache/beam/dsls/sql/BeamSqlDslAggregationTest.java

```diff
@@ -257,4 +257,31 @@ public void testSessionWindow() throws Exception {
 
     pipeline.run().waitUntilFinish();
   }
+
+  @Test
+  public void testWindowOnNoneTimestampField() throws Exception {
```

Comment:
s/None/Non/ 

As in "non-timestamp field", which I believe is what you meant here.